src/main/java/com/gtnewhorizons/modularui/common/internal/wrapper/ModularGui.java

```diff
@@ -482,10 +484,12 @@ protected void mouseClicked(int mouseX, int mouseY, int mouseButton) {
                 // if floating window is clicked (while holding item), widgets/slots below should not be interacted
                 probablyClicked = hovered;
                 wasReject = true;
+                window = mainWindow;
```

Comment:
Did you mean `hovered` here? This assignment does nothing

Reply:
Ah nvm, this is in a loop, the review was confusing without context

Reply:
yeah. thanks for catching this.